src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/LegacyRestrictionType.kt

```diff
@@ -8,20 +8,5 @@ package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto
 enum class LegacyRestrictionType(val description: String) {
   CELL("Do Not Locate in Same Cell"),
   LAND("Do Not Locate on Same Landing"),
-  NONEX("Do Not Exercise Together"),
```

Comment:
What happened to these restriction types?

Reply:
They don't exist in NOMIS and we never convert them back